ts/receiver/conversation.d.ts

```diff
@@ -0,0 +1,39 @@
+interface ConversationAttributes {
```

Comment:
for this i meant put it in
`js/models/conversation.d.ts`

Reply:
moved